### ISSUE TITLE
BUG: New node button changes size after beginning to type

### ISSUE THREAD

**Issue by jdanish** [netcreateorg/netcreate-itest#380]
Load a new network, and new node takes up 2 lines:

<img width="355" alt="Image" src="https://github.com/user-attachments/assets/233d18a6-c392-4f4a-bb66-bce24d30875a" />

Begin typing and it re-sizes in addition to changing color. The original should be the same as the highlighted I assume?
